packages/devtools_app/lib/src/screens/inspector_v2/inspector_controller.dart

```diff
@@ -201,9 +200,14 @@ class InspectorController extends DisposableController
   ValueListenable<InspectorTreeNode?> get selectedNode => _selectedNode;
   final _selectedNode = ValueNotifier<InspectorTreeNode?>(null);
 
-  ValueListenable<List<RemoteDiagnosticsNode>> get selectedNodeProperties =>
-      _selectedNodeProperties;
-  final _selectedNodeProperties =
+  ValueListenable<List<RemoteDiagnosticsNode>>
+      get selectedNodeWidgetProperties => _selectedNodeWidgetProperties;
+  final _selectedNodeWidgetProperties =
+      ValueNotifier<List<RemoteDiagnosticsNode>>([]);
+
+  ValueListenable<List<RemoteDiagnosticsNode>>
+      get selectedNodeRenderProperties => _selectedNodeRenderProperties;
+  final _selectedNodeRenderProperties =
```

Comment:
nit: can we use a pattern to store the widget properties and the render object properties as a pair so that we only have to update a single notifier? This pattern can be stored as a typedef InspectorNodeProperties or something like this

Reply:
Done! Using a record to store them